quiz.css

```diff
@@ -0,0 +1,68 @@
+article {
+	width: 30%;
+	float: left;
+	border: 1px solid lightblue;
+	margin: 1.5em 1em .5em 1em;
+}
+
+h1 {
+	border: 1px dotted lightgrey;
+	text-align: center;
+	margin-top: 0;
+	margin-bottom: 0;
+	padding-bottom: 1em;
+	padding-top: .7em;
+	height: 2em;
+}
+
+.cerimg {
+	padding-top: .5em;
+}
+.descript {
+	padding: .5em .5em 0 .5em;
+	text-align: justify;
+}
+
+.onstock {
+	color: yellow;
+	background-color: grey;
+	margin-top: .5em;
+	padding-bottom: .2em;
+	padding-top: .2em;
+}
+
+img {
+	width: 350px;
+	height: 350px;
+	display: block;
+	margin: 0 auto 0 auto;
+}
+
+h4 {
+	font-family: Arial, sans-serif, serif;
+	text-indent: .5em;
+	margin-bottom: 0;
+	font-weight: 600;
+}
+
+.priceweight p {
+	font-family: Arial, sans-serif;
+	font-size: .8em;
+	text-indent: 1em;
+	margin: 0;
+	font-weight: bolder;
+}
+
+.specfut {
+	font-size: .6em;
+	margin: 0;
+}
+
+.spcup  {
+	margin-top: 0;
+}
+
+h1 {
```

Comment:
This is the second time you've defined an `h1` in the styles. You should combine them.